src/v/redpanda/application.cc

```diff
@@ -1114,6 +1114,9 @@ void application::wire_up_runtime_services(
           ss::sharded_parameter([this] {
               return transform::rpc::partition_manager::make_default(
                 &shard_table, &partition_manager);
+          }),
+          ss::sharded_parameter([this] {
+              return transform::service::create_reporter(&_transform_service);
```

Comment:
regarding the use of sharded_parameter, i don't see anything in `create_reporter` that depends on core-local state. am i missing something?

Reply:
`create_reporter` returns a `std::unique_ptr` so we want to allocate that on each shard and there is no easy way to copy that.